packages/sycamore-reactive/src/signal.rs

```diff
@@ -566,6 +566,23 @@ impl<T: Display> Display for ReadSignal<T> {
     }
 }
 
+/* Into<bool> implementation */
+
+impl From<RcSignal<bool>> for bool {
+    fn from(signal: RcSignal<bool>) -> bool {
+        *signal.get()
+    }
+}
+impl From<&Signal<bool>> for bool {
+    fn from(signal: &Signal<bool>) -> bool {
+        *signal.get()
+    }
+}
+impl From<&ReadSignal<bool>> for bool {
+    fn from(signal: &ReadSignal<bool>) -> bool {
+        *signal.get()
+    }
+}
```

Comment:
You're right. I thought it was required but I can actually just put the whole expression in a closure and it'll work with the current syntax. I still think it should be added specifically to allow third party code to accept both signals and non-reactive values as a single prop and it also makes sense since `Display` and `Hash` are implemented, but like you said that's a separate PR.